synapse/handlers/device.py

```diff
@@ -157,7 +157,7 @@ async def get_user_ids_changed(
             # The user may have left the room
             # TODO: Check if they actually did or if we were just invited.
             if room_id not in room_ids:
-                for key, event_id in current_state_ids.items():
+                for key, _ in current_state_ids.items():
```

Comment:
Looks like this could also use `current_state_Ids.keys()`.

Reply:
oh whoops, i did that a few more times, i didn't see this was an `.items` call, good catch :+1: